src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,29 @@
+import classNames from 'classnames';
+import React from 'react';
+type Props = {
+  errorMessage: string;
+  onError: (er: string) => void;
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  errorMessage,
+  onError = () => {},
```

Comment:
```suggestion
  onError,
```
`onError` is required so you don't need to set a default value